src/item_factory.cpp

```diff
@@ -1422,6 +1444,8 @@ void Item_factory::load_basic_info(JsonObject &jo, itype *new_item_template)
         new_item_template->category = get_category( jo.get_string( "category" ) );
     }
 
+    assign( jo, "faults", new_item_template->faults );
```

Comment:
YAGNI here, it's the same concept, don't make something a general
capability if it's only used in one place.